pnpm-lock.yaml

```diff
@@ -2839,6 +2876,9 @@ packages:
   '@types/node@20.17.16':
     resolution: {integrity: sha512-vOTpLduLkZXePLxHiHsBLp98mHGnl8RptV4YAO3HfKO5UHjDvySGbxKtpYfy8Sx5+WKcgc45qNreJJRVM3L6mw==}
 
+  '@types/node@20.17.22':
+    resolution: {integrity: sha512-9RV2zST+0s3EhfrMZIhrz2bhuhBwxgkbHEwP2gtGWPjBzVQjifMzJ9exw7aDZhR1wbpj8zBrfp3bo8oJcGiUUw==}
+
```

Comment:
Type checking is failing again because there are two different `@types/node` versions being used as peer deps. I wonder if upgrading pnpm to v10 wouldn't alleviate some of these issues.

Reply:
I just reverted pnpm-lock.yaml changes and re-ran `pnpm i` manually to get rid of all the extra updates. I wonder if this happens because dependabot doesn't have access to teleport.e and pnpm behaves differently when we patch the lockfile.

Reply:
Cloning the repo at aa3faf9 without submodules and then running `pnpm i` doesn't produce those weird peer dep changes, so it's likely dependabot that's misbehaving.